### DISCUSSION TITLE
XHTTP STREAM-UP/ONE and QUIC/XMUX

### DISCUSSION THREAD

**Discussion by PentiumB** [lxhao61/integrated-examples#226] [Q&A]
hi. How about examples of configurations with Stream-One (\up) with splitting downstream and upstream and splitting them into h2 and h3. And using this configuration for cdn?
I would also like to look at QUIC/XMUX .

**Comment by PentiumB** [comment]
source: https://github.com/XTLS/Xray-core/discussions/4113

**Comment by lxhao61** [comment]
见本仓库中 XHTTP 应用（包括综合应用中 XHTTP 应用）。
上下行分离主要由客户端配置。
